Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "rust-g"
 edition = "2018"
-version = "0.7.0"
+version = "0.7.1"
```

Comment:
Don't change the version in this PR (also this would be 0.8.0 anyway)

Reply:
sure